pkg/drawio/subnetsLayout.go

```diff
@@ -465,9 +465,16 @@ func (ly *subnetsLayout) newGroupFromSplitMiniGroups(group *groupDataS, miniGrou
 
 func (ly *subnetsLayout) calcZoneOrder() {
 	zonesScores := ly.calcZonePairScores()
-	zoneOrder := []TreeNodeInterface{}
+	var zoneOrder []TreeNodeInterface
+	zoneOrders := [][]TreeNodeInterface{}
 	for len(zonesScores) > 0 {
-		zoneToAdd, addToRight := chooseZoneToAdd(zonesScores, zoneOrder)
+		zoneToAdd, addToRight, newZoneOrder := chooseZoneToAdd(zonesScores, zoneOrder)
+		if newZoneOrder {
+			if zoneOrder != nil {
```

Comment:
```suggestion
			if len(zoneOrder) > 0 {
```
Looks safer, and consistent with the condition after the end of the loop

Reply:
done